test/test_cuda_mapper.cc

```diff
@@ -851,6 +851,41 @@ def fun(float(N) I) -> (O) {
   EXPECT_TRUE(code.find("O[0] = (O") != std::string::npos);
 }
 
+/*
+ * Check that a 2D mean produce a reduction library call.
```

Comment:
I think something is missing after "2d mean", perhaps "TC"?
Also "produce" -> "produces".